assets/base.css

```diff
@@ -3125,7 +3129,8 @@ details-disclosure > details {
   .select__select:focus,
   .customer .field input:focus,
   .customer select:focus,
-  .localization-form__select:focus.localization-form__select:after {
+  .localization-form__select:focus.localization-form__select:after,
+  .country-filter__input:focus {
```

Comment:
Here I feel like we don't really need any of those `country-filter__input` added to those CSS declarations since it's already getting it from `field__input` no ? Or am I missing something 🤔

Reply:
Yep you're 100% right. Initially I wasn't using `field__input` and I just never thought of removing these once I did start using them.